blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How a hack week project made Sourcegraph.com easier to use, by searching the top 10k open source repositories on GitHub by default."
```

Comment:
You mention hack week in the description but there is no mention of this in the body. I would remove this from the description or discuss it in the body.

Reply:
Agreed and removed.